src/CrudPanel.php

```diff
@@ -226,4 +226,27 @@ public function sort($items)
 
         return $this->{$items};
     }
+
+    /**
+     * Get the Eloquent Model name from the given relation definition string.
+     *
+     * @example For a given string 'company' and a relation between App/Models/User and App/Models/Company, defined by a
+     *          company() method on the user model, the 'App/Models/Company' string will be returned.
+     *
+     * @example For a given string 'company.address' and a relation between App/Models/User, App/Models/Company and
+     *          App/Models/Address defined by a company() method on the user model and an address() method on the
+     *          company model, the 'App/Models/Address' string will be returned.
+     *
+     * @param $relationString String Relation string. A dot notation can be used to chain multiple relations.
+     *
+     * @return string relation model name
+     */
+    private function getRelationModel($relationString)
+    {
+        $result = array_reduce(explode('.', $relationString), function ($obj, $method) {
+            return $obj->$method()->getRelated();
```

Comment:
That could easily explode unexpectedly if $method doesn't exist, $obj isn't an object or such.

What should it do though:

1. Explode with PHP messiness
2. Be coded defensively and do something else
3. Be put into a try catch

Something else?